web-client/public/js/setup-handlers.js

```diff
@@ -20,18 +22,28 @@ export const setupHandlers = grnState => {
         updateApp(grnState);
     });
 
-    $("#normalization-button").click(function () {
+    $("#normalization-button").click(() => {
         grnState.normalizationMax = $("#normalization-max").val();
         updateApp(grnState);
     });
 
-    $("#reset-normalization-factor-menu, #resetNormalizationButton").click(function () {
+    $("#reset-normalization-factor-menu, #resetNormalizationButton").click(() => {
         grnState.normalizationMax = grnState.resetNormalizationMax;
         updateApp(grnState);
     });
 
-    $("#edge-weight-normalization-factor-menu").on("change", function () {
+    $("#edge-weight-normalization-factor-menu").change(() => {
         grnState.normalizationMax = $("#edge-weight-normalization-factor-menu").val();
         updateApp(grnState);
     });
+
+    $(GREY_EDGE_THRESHOLD_MENU).change(() => {
+        grnState.grayEdgeThreshold = Math.round(($(GREY_EDGE_THRESHOLD_MENU).val()));
+        updateApp(grnState);
+    });
+
+    $(GREY_EDGE_THRESHOLD_SLIDER_SIDEBAR).change(() => {
+        grnState.grayEdgeThreshold = Math.round(($(GREY_EDGE_THRESHOLD_SLIDER_SIDEBAR).val() * 100));
```

Comment:
Same here, I don't think you need the one immediately inside the `round()` argument.